content/docs/introduction/comparison.md

```diff
@@ -246,31 +246,30 @@ environment, then Prometheus is a good choice.
 
 ## Prometheus vs. Sensu
 
-[Sensu](https://sensu.io) is a composable monitoring pipeline that can reuse existing Nagios checks.
+[Sensu](https://sensu.io) is an open source monitoring and observabliity pipeline with a commercial distribution. It can reuse existing Nagios plugins for compatibility.
 
 ### Scope
 
-The same general scope differences as in the case of [Nagios](/docs/introduction/comparison/#prometheus-vs-nagios) apply here.
-
-There is also a [client socket](https://docs.sensu.io/sensu-core/latest/reference/clients/#what-is-the-sensu-client-socket) permitting ad-hoc check results to be pushed into Sensu. 
+Sensu processes events via the [Sensu Observablity Pipeline](https://docs.sensu.io/sensu-go/latest/observability-pipeline/), where events are deduplicated/correlated (via [EventFilters](https://docs.sensu.io/sensu-go/latest/observability-pipeline/observe-filter/)), transformed (via [Mutators](https://docs.sensu.io/sensu-go/latest/observability-pipeline/observe-transform/)), and processed (via [Handlers](https://docs.sensu.io/sensu-go/latest/observability-pipeline/observe-process/)). Sensu also collects and routes metrics to a number of backend time-series databases such as Prometheus, InfluxDB, etc.
 
 ### Data model
 
-Sensu has the same rough data model as [Nagios](/docs/introduction/comparison/#prometheus-vs-nagios).
+Sensu primarily focuses on normalizing and processing observability data as [Events](https://docs.sensu.io/sensu-go/latest/observability-pipeline/observe-events/events/), identified by an event name, entity (e.g. node, container, or service), and optional [key-value metadata](https://docs.sensu.io/sensu-go/latest/observability-pipeline/observe-events/events/#metadata-attributes) called "labels" or "annotations". The Sensu agents can also extract and route metrics from common formats such as Prometheus Exposition Format, InfluxDB line protocal, Graphite plain text, OpenTSDB line protocol, StatsD, and Nagios Perfdata and rewrite them into any backend storage format (Elastic, Influx, Wavefront, Prometheus).
 
 ### Storage
 
-Sensu uses Redis to persist monitoring data, including the Sensu client registry, check results, check execution history, and current event data.
+Sensu _caches_ event status information and real-time inventory data in an embedded database (etcd) or an external RDBMS (Postgres). All event and time series data is sent to one or more specialized datastores for long term storage and analysis (e.g. Prometheus, InfluxDB, Wavefront, TimescaleDB, Elasticsearch, Splunk).
```

Comment:
That smells like it might be more like storage than a cache to me. If the cache was flushed or down, would that affect results?